pkg/capabilities/datastreams/types.go

```diff
@@ -73,6 +73,12 @@ type SignersMetadata struct {
 	MinRequiredSignatures int
 }
 
+type StreamsTriggerPayload struct {
```

Comment:
"Payload" in the name is a little confusing. How about "StreamsTriggerEvent"? The line of thinking being Event=Payload+Metadata.